packages/transport/test/modules/files/FileController.test.ts

```diff
@@ -154,4 +154,115 @@ describe("FileController", function () {
+            expect(result.isValid).toBe(true);
+        });
+
+        test("should not validate an ownershipToken as not the owner", async function () {
+            await TestUtil.expectThrowsRequestErrorAsync(recipient.files.validateFileOwnershipToken(senderFile.id, ownershipToken), "error.platform.unexpected", 403);
+        });
+
+        test("should regenerate an ownershipToken as the owner", async function () {
+            const updatedFile = await sender.files.regenerateFileOwnershipToken(senderFile.id);
+            expect(updatedFile.ownershipToken).toBeDefined();
+            expect(updatedFile.ownershipToken).not.toBe(ownershipToken);
+        });
+
+        test("should not regenerate an ownershipToken as not the owner", async function () {
+            await TestUtil.expectThrowsRequestErrorAsync(recipient.files.regenerateFileOwnershipToken(senderFile.id), "error.platform.unexpected", 403);
+        });
+
+        test("should fetch a File after the ownership was transferred as the previous owner", async function () {
+            await recipient.files.getOrLoadFile(senderFile.id, senderFile.secretKey);
+            await recipient.files.claimFileOwnership(senderFile.id, ownershipToken);
+
+            const [fetchedFile] = await sender.files.updateCache([senderFile.id.toString()]);
+            expect(fetchedFile.isOwn).toBe(false);
+            expect(fetchedFile.cache!.owner).toStrictEqual(recipient.identity.address);
+        });
+
+        test("should claim the ownership of a File with a valid ownershipToken as not the owner after loading it", async function () {
+            await recipient.files.getOrLoadFile(senderFile.id, senderFile.secretKey);
+
+            const updatedFile = await recipient.files.claimFileOwnership(senderFile.id, ownershipToken);
+            expect(updatedFile.cache!.owner).toStrictEqual(recipient.identity.address);
+            expect(updatedFile.isOwn).toBe(true);
+
+            expect(updatedFile.ownershipToken).toBeDefined();
+            expect(updatedFile.ownershipToken).not.toBe(ownershipToken);
+        });
+
+        test("should claim the ownership of a File with a valid ownershipToken as the owner after loading it", async function () {
+            const updatedFile = await sender.files.claimFileOwnership(senderFile.id, ownershipToken);
+            expect(updatedFile.cache!.owner).toStrictEqual(sender.identity.address);
+            expect(updatedFile.isOwn).toBe(true);
+
+            expect(updatedFile.ownershipToken).toBeDefined();
+            expect(updatedFile.ownershipToken).not.toBe(ownershipToken);
+        });
+
+        test("should not claim the ownership of a File with an invalid ownershipToken", async function () {
+            await TestUtil.expectThrowsRequestErrorAsync(recipient.files.claimFileOwnership(senderFile.id, "invalid-token"), "error.platform.unexpected", 403);
+        });
```

Comment:
Writing `"anInvalidOwnershipToken"` (or `"anInvalidToken"`) should match our test value naming convention better than using `"invalid-token"`. Similar for other tests.

```suggestion
        test("should not claim the ownership of a File with an invalid ownershipToken", async function () {
            await TestUtil.expectThrowsRequestErrorAsync(recipient.files.claimFileOwnership(senderFile.id, "anInvalidOwnershipToken"), "error.platform.unexpected", 403);
        });
```